Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -152,12 +197,55 @@ QTabBar::tab:focus {
     margin-left: 50px;
 }
 
+#createAGemFormErrorLabel {
+    color: #ec3030;
+    font-size: 14px;
+}
+
 #formTitleLabel {
     font-size:21px;
     color:#ffffff;
     margin: 24px 0 10px 50px;
 }
 
+#rightPaneHeader {
+    font-size:21px;
+    qproperty-indent: 0;
+}
+
+#rightPaneSubheader {
+    font-size:14px;
+    qproperty-indent: 0;
+}
+
+
+#createAGemRightPane {
+    padding-top: 25px;
+    padding-left: 60px;
+}
+
+#leftPaneText {
+    font-size: 16px;
+}
+
+#qRadioButton {
+    padding-top: 15px;
+    font-size: 14px;
+}
```

Comment:
```suggestion
#createAGem #QRadioButton{
    padding-top: 15px;
    font-size: 14px;
}
```
You can use the existing object name instead of defining your own

Reply:
Fixed.